src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -46,7 +42,14 @@ private static void setupPresetTests(Application application) {
 	 */
 	private static void setupCommandTests(Application application) {
 		application.addTest("Load secret command", new SelectLoadSecretCommandOptionListener());
-
+		application.addTest("Test complex command builder", new SelectLoadComplexCommandListener(
+				new ComplexCommandBuilder()
+						.addSetPosition(0, 0)
+						.addOperateTo(10, 0)
+						.addOperateTo(10, 10)
+						.addOperateTo(0, 10)
+						.addOperateTo(0, 0)
+							.build()));
```

Comment:
extract to separate class (and method). Give a better name for the test (e.g., "Load small rectangle").